code/modules/experisci/experiment/experiments.dm

```diff
@@ -321,11 +321,11 @@
 	. = ..()
 	damage_percent = rand(15, 95)
 	//updating the description with the damage_percent var set
-	description = "Your exosuit fabricators allow for rapid production on a small scale, but the structural integrity of created parts is inferior to those made with more traditional means. Damage a few exosuits to exactly [damage_percent]% integrity and scan them to help us determine how the armor fails under stress."
+	description = "Your exosuit fabricators allow for rapid production on a small scale, but the structural integrity of created parts is inferior to those made with more traditional means. Damage a few exosuits to around [damage_percent]% integrity and scan them to help us determine how the armor fails under stress."
 
 /datum/experiment/scanning/random/mecha_damage_scan/final_contributing_index_checks(atom/target, typepath)
-	var/found_percent = round(target.get_integrity() / target.max_integrity, 0.01) * 100
-	return ..() && (found_percent == damage_percent)
+	var/found_percent = round((target.get_integrity() / target.max_integrity) * 100)
+	return ..() && (found_percent <= (damage_percent + 2) && found_percent >= (damage_percent - 2))
```

Comment:
Does this even work? It's using ``&&``, so if the damage is 2 above, because it isn't also 2 below, it should be returning false?
```suggestion
	return ..() && ((found_percent <= (damage_percent + 2) || found_percent >= (damage_percent - 2)))
```

Reply:
It has to be above x-2 and below x+2, as in x-2 < y < x+2. It works.

Changing the operator to "or" would make it so it has to be less than x+2 or greater than x-2, which encompasses every real number. y < x+2 encompasses negative infinity to x+2, and y > x-2 encompasses x-2 to positive infinity. What we want is the intersection of these sets, not their union.

I just opened up a private server and tested it again to make sure it's working as intended, and it is. With a target of 73%, it accepted at 73% and 75% but not at 90%.

Reply:
You're right, I misread it entirely.